simba_ml/prediction/export.py

```diff
@@ -23,33 +23,43 @@ def export_input_batches(
             data[i],
             columns=input_features,
         ).to_csv(
-            os.path.join(os.getcwd(), export_path, f"input_{i}.csv"),
+            os.path.join(os.getcwd(), export_path, f"X_test_{i}.csv"),
```

Comment:
From the name of the function, I would not expect the written data to only be test data.